src/amp-story-ad-preview.js

```diff
@@ -62,8 +62,13 @@ const httpsCircumventionPatch = minifyInlineJs(`
   })();
 `);
 
-const insertHttpsCircumventionPatch = docStr =>
-  docStr.replace('<head>', `<head><script>${httpsCircumventionPatch}</script>`);
+const insertPatches = docStr => {
+  let str = docStr.replace(
+    '<head>',
+    `<head><script>${httpsCircumventionPatch}</script>`
+  );
+  return str.replace('<body>', '<body amp-story-visible>');
```

Comment:
This will only work if `<body>` has no attributes!

Let's use a regex:

```js
const setBodyAmpStoryVisible = docStr =>
  docStr.replace(/<(body[^>]*)>/, '<$1 amp-story-visible>');
```

(in the replacement, `$1` is the first group match, so `body` or `body class="foo"` etc.)